BrainPortal/db/schema.rb

```diff
@@ -117,6 +117,35 @@
   add_index "data_providers", ["type"], :name => "index_data_providers_on_type"
   add_index "data_providers", ["user_id"], :name => "index_data_providers_on_user_id"
 
+  create_table "demands", :force => true do |t|
```

Comment:
This is a spurious table specification. Please make sure your dev DB is in sync with the schema.